lib/oli_web/views/layout_view.ex

```diff
@@ -131,4 +132,58 @@ defmodule OliWeb.LayoutView do
   defp show_collab_space?(nil), do: false
   defp show_collab_space?(%CollabSpaceConfig{status: :disabled}), do: false
   defp show_collab_space?(_), do: true
+
+  def render_license(%{license: %{license_type: :custom} = license} = assigns) do
+    assigns = Map.put(assigns, :custom_license_details, license.custom_license_details)
+
+    ~H"""
+    <div
+      id="license"
+      class="container mx-auto flex flex-row px-10 items-center overflow-y-auto h-[40px] max-h-[40px] relative top-0"
+    >
+      <p class="h-full">License: <%= @custom_license_details %></p>
+    </div>
+    """
+  end
+
+  def render_license(%{license: %{license_type: cc_license}} = assigns) do
+    cc_data =
+      Enum.find(CreativeCommons.cc_options(), fn cc_option -> cc_option.id == cc_license end)
+
+    logo_name =
+      cc_data.id |> Atom.to_string() |> String.replace("cc_", "") |> String.replace("_", "-")
+
+    cc_text = cc_data |> Map.get(:text) |> String.split(":") |> Enum.at(1)
+
+    assigns =
+      assigns
+      |> Map.put(:cc_data, cc_data)
```

Comment:
I think we do not need to assign `cc_data` as we do not use it in the template.

Reply:
Good catch!